app.ts

```diff
@@ -206,7 +206,7 @@ if (defArgs.wantedLanguage) {
     languages = filteredLangs;
 }
 
-if (languages.length === 0) {
+if (_.isEmpty(languages)) {
```

Comment:
oh cripes there's going to be a lot of these!

Reply:
this is specifically a bug in the original implementation that was discovered because typescript started complaining about it.

`languages` is an object, not a list, and so it doesn't have a `length` parameter